<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>schedule_manage</class>
 <widget class="QMainWindow" name="schedule_manage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Schedule Manage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Schedules</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_schedule">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>841</width>
      <height>291</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>ID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Doctor</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Patient</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Result</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Edit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Delete</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_import_schedule">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>80</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_export_schedule">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>941</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>851</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Search schedule</string>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_search_schedule">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>651</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date From:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>520</x>
         <y>100</y>
         <width>91</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Date To:</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="data_from_search_schedule">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>91</y>
         <width>194</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="date_to_search_schedule">
       <property name="geometry">
        <rect>
         <x>600</x>
         <y>90</y>
         <width>194</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_search_schdule">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>160</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Search Schedule</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Add new</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Doctor ID:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Patient ID:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_name_schedule">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>721</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_add_schedule">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>230</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="date_add_schedule">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_doctor_schedule">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_patient_schedule">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>170</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>986</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
